src/Components/VoteCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react';
+import { patchVotes } from '../../api';
+
+const VoteCard = ({ article }) => {
+    
+    const [voteCount, setVoteCount] = useState(article.votes);
+
+    const positiveHandler = () => {
+     setVoteCount((currentVotes) => {
+        return currentVotes + 1
+     })
+     patchVotes(article.article_id, 1)
+    };
+
+   const negativeHandler = () => {
+    setVoteCount((currentVotes) => {
+        return currentVotes - 1
+    })
+    patchVotes(article.article_id, -1)
+   }
+
```

Comment:
This works great and nice use of optimistic rendering. However, what if something went wrong and the patch request failed? For example if the internet went off! Currently, the user will think that the vote was successful. Have a go at implementing logic that will put the vote number back to what it was if it failed along with some kind of error message 🙂